MODULE.bazel

```diff
@@ -1,6 +1,146 @@
-###############################################################################
-# Bazel now uses Bzlmod by default to manage external dependencies.
-# Please consider migrating your external dependencies from WORKSPACE to MODULE.bazel.
-#
-# For more details, please check https://github.com/bazelbuild/bazel/issues/18958
-###############################################################################
+module(name = "rules_scala_annex")
+
+bazel_dep(name = "bazel_skylib", version = "1.7.1")
+
+# TODO: Use buildifier instead of buildifier_prebuilt once
+# https://github.com/bazelbuild/bazel-central-registry/issues/380 is resolved
+bazel_dep(name = "buildifier_prebuilt", version = "7.3.1")
```

Comment:
Can we mark dev dependencies as such, so they don't propagate to downstream projects? I believe we can just set `dev_dependency = True` on the `bazel_dep` call to make that the case.

Reply:
I originally tried marking this as a dev dependency, but then the test repository failed to build because we reference `buildifier_prebuilt` in `@rules_scala_annex//:BUILD.bazel`.